android-components/components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/manager/FxaAccountManager.kt

```diff
@@ -449,10 +455,16 @@ open class FxaAccountManager(
     internal suspend fun encounteredAuthError(
         operation: String,
         errorCountWithinTheTimeWindow: Int = 1,
-    ) = withContext(coroutineContext) {
-        processQueue(
-            Event.Account.AuthenticationError(operation, errorCountWithinTheTimeWindow),
-        )
+    ) {
+        if (authRecoveryScheduled.getAndSet(true)) {
+            crashReporter?.recordCrashBreadcrumb(Breadcrumb("multiple auth recoveries scheduled at once"))
```

Comment:
How do we know that multiple auth recoveries were scheduled at this point?  The `encounteredAuthError` function is the only place where `authRecoveryScheduled` is set to true. Also, should we rename it `authRecoveryScheduled` to `multipleAuthRecoveryScheduled` if it's the intention of the flag?